### DISCUSSION TITLE
Verify all images at once in frigate+

### DISCUSSION THREAD

**Discussion by aaronjrodrigues** [blakeblackshear/frigate#14171] [Ask A Question]
I wanted to ask if it is possible as I have like 300 images I need to click verify even though I've already had a look at them when using the snapshots view in frigate.

**Comment by blakeblackshear** [comment]
It's not possible. Keep in mind that submitting the image from Frigate only submits information about one potential label in the image. It would be unusual for an image to not include anything else. Images need to have **all** objects labeled. Here is a snippet from this page in the docs: https://docs.frigate.video/plus/improving_model#properly-labeling-images

> For the best results, follow the following guidelines.
> 
> **Label every object in the image**: It is important that you label all objects in each image before verifying. If you don't label a car for example, the model will be taught that part of the image is _not_ a car and it will start to get confused.
> 
> **Make tight bounding boxes**: Tighter bounding boxes improve the recognition and ensure that accurate bounding boxes are predicted at runtime.
> 
> **Label the full object even when occluded**: If you have a person standing behind a car, label the full person even though a portion of their body may be hidden behind the car. This helps predict accurate bounding boxes and improves zone accuracy and filters at runtime. If an object is partly out of frame, label it only when a person would reasonably be able to recognize the object from the visible parts.
> 
> **Label objects hard to identify as difficult**: When objects are truly difficult to make out, such as a car barely visible through a bush, or a dog that is hard to distinguish from the background at night, flag it as 'difficult'. This is not used in the model training as of now, but will in the future.
> 
> **`amazon`, `ups`, and `fedex` should label the logo**: For a Fedex truck, label the truck as a `car` and make a different bounding box just for the Fedex logo. If there are multiple logos, label each of them.

**Reply by sarahbau** [reply]
I don't think it's _that_ unusual of an occurrence, at least as I'm still getting things set up. Something I've done on multiple occasions now, is see an animal I hadn't seen before that gets misidentified (say a raccoon getting identified as a dog). So I want to add raccoon as an option to all of my cameras. That means I have to go through every image I've already labeled on every camera and say "there are no raccoons here" even though I know this is the first raccoon I've seen.

**Reply by blakeblackshear** [reply]
You don't have to go back through those. As long as they have been previously verified against prior labels, they are still included in training. Going back through and checking all of them for raccoons would not change anything about the images included for training for your next model request if there aren't any raccoons in them.
